[ENG-1742] Add pentesting docs (#16)
Oneleet/oneleet-docs: pages/penetration-testing/ptaas.mdx

# Penetration Testing as a Service (PtaaS) at Oneleet

## About Us

Oneleet is a United States-based cybersecurity company, established and ran by experienced penetration testers. The company offers flexible penetration testing options and a comprehensive platform for managing and addressing security vulnerabilities. Its interface facilitates the tracking and remediation of security findings, ensuring that organizations maintain robust and current defenses. Oneleet provides both expert testing services and a management system to facilitate the maintenance and enhancement of security posture. The company has received backing from venture capital firms such as Y Combinator positioning itself as a key competitor that prioritizes support, effectiveness, and communication. Oneleet serves a diverse clientele, ranging from enterprises to early-stage startups.

## Our Penetration Testing Goal

> Identifying vulnerabilities to reduce risk. Simulating real world attacks on your applications, systems and networks.
> 

> The primary objective of a penetration testing at Oneleet is to identify vulnerabilities before malicious actors exploit them, thereby fortifying your security program. We are excited to collaborate with you in your commitment to uncovering vulnerabilities and implementing robust protection measures.

Reviewer: ```suggestion
The primary objective of a penetration test at Oneleet is to identify vulnerabilities before malicious actors exploit them, thereby fortifying your security program. We are excited to collaborate with you in your commitment to uncovering vulnerabilities and implementing robust protection measures.
```

> The primary objective of a penetration testing at Oneleet is to identify vulnerabilities before malicious actors exploit them, thereby fortifying your security program. We are excited to collaborate with you in your commitment to uncovering vulnerabilities and implementing robust protection measures.

Reviewer: ```suggestion
The primary goal of penetration testing at Oneleet is to uncover vulnerabilities before they can be exploited by malicious actors. We look forward to partnering with you to identify risks and implement effective protection measures.
```

## Services

> Oneleet offers expertly conducted Penetration Testing services by our team of highly qualified professionals from NATO countries. They hold advanced certifications like OSCP and OSCE or OSWE. Our team’s expertise encompasses network penetration (wired and wireless), web and mobile application security, social engineering, and code reviews. This extensive skill set enables them to identify vulnerabilities across various systems and technologies.

Reviewer: Oneleet provides expertly conducted penetration testing services, delivered by a team of highly qualified professionals from NATO countries. Our experts hold advanced certifications such as OSCP, OSCE, and OSWE, attesting to their high level of technical competance. Their expertise spans network penetration (wired and wireless), web and mobile application security, social engineering, and code reviews. This diverse skill set allows them to identify vulnerabilities across a wide range of systems and technologies.

> Oneleet offers expertly conducted Penetration Testing services by our team of highly qualified professionals from NATO countries. They hold advanced certifications like OSCP and OSCE or OSWE. Our team’s expertise encompasses network penetration (wired and wireless), web and mobile application security, social engineering, and code reviews. This extensive skill set enables them to identify vulnerabilities across various systems and technologies.

Reviewer: ```suggestion
Oneleet provides expertly conducted penetration testing services, delivered by a team of highly qualified professionals from NATO countries. Our experts hold advanced certifications such as OSCP, OSCE, and OSWE, attesting to their high level of technical competance. Their expertise spans network penetration (wired and wireless), web and mobile application security, social engineering, and code reviews. This diverse skill set allows them to identify vulnerabilities across a wide range of systems and technologies.
```

> We provide flexible retesting options as part of our standard penetration testing package and offer a comprehensive platform for managing vulnerabilities.

Reviewer: ```suggestion
We offer flexible retesting options as part of our standard penetration testing package, along with a comprehensive platform for managing vulnerabilities.
```

> At Oneleet, we frequently conduct penetration tests to meet compliance requirements for frameworks like SOC 2, ISO 27001, PCI, HIPAA and more.

Reviewer: ```suggestion
At Oneleet, we frequently conduct penetration tests to help organizations meet compliance requirements for frameworks like SOC 2, ISO 27001, PCI DSS, HIPAA, and more.
```
